netci.groovy

```diff
@@ -59,10 +59,15 @@ platformList.each { platform ->
                 dockerContainer = "ubuntu-16.04-cross-arm64-a3ae44b-20180316023254"
             }
             else {
-                dockerContainer = "ubuntu-14.04-cross-0cd4667-20172211042239"
+                dockerContainer = "ubuntu-14.04-cross-e435274-20180323032140"
             }
             dockerCommand = "docker run -e ROOTFS_DIR=/crossrootfs/${architecture} --name ${dockerContainer} --rm -v \${WORKSPACE}:${dockerWorkingDirectory} -w=${dockerWorkingDirectory} ${dockerRepository}:${dockerContainer}"
-            buildArgs += " -SkipTests=true -DisableCrossgen=true -CrossBuild=true"
+            buildArgs += " -SkipTests=true -CrossBuild=true"
+
+            if (architecture == 'armel' || architecture == 'arm64') {
```

Comment:
Why do we disable it for armel? Just being curious.

Reply:
It was disabled before, so I wasn't changing that.

I'm not sure when and where we build for `linux-armel`.  The Tizen case doesn't get into this, so I kind of think that `armel` in this code is a dead code path.  We don't ever have os = "Linux" and architecture = "armel".